packages/clay-label/src/ClayLabel.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import * as React from 'react';
+import classNames from 'classnames';
+import Icon from '@clayui/icon';
+
+type DisplayType = 'secondary' | 'info' | 'warning' | 'danger' | 'success';
+
+interface Props
+	extends React.BaseHTMLAttributes<HTMLAnchorElement | HTMLSpanElement> {
+	closeButtonProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
```

Comment:
I had forgotten about this, but wanted us to continue adding the API descriptions here.

Reply:
oh good call, I had forgotten too. I'll send a new PR to update all the existing attributes that were merged and then I will update this PR and any others that I have open.

Reply:
Okay, thanks Bryce! I'll update the clay-link as well.